nucypher/cli/painting.py

```diff
@@ -854,41 +855,47 @@ def echo_solidity_version(ctx, param, value):
 
 def paint_worklock_status(emitter, registry: BaseContractRegistry):
     from maya import MayaDT
-    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)  # type: WorkLockAgent
     blockchain = worklock_agent.blockchain
 
-    # Agency
-    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=registry)
-
     # Time
-    start = MayaDT(worklock_agent.contract.functions.startBidDate().call())
-    end = MayaDT(worklock_agent.contract.functions.endBidDate().call())
-    duration = end - start
-    remaining = end - maya.now()
+    bidding_start = MayaDT(worklock_agent.contract.functions.startBidDate().call())
+    bidding_end = MayaDT(worklock_agent.contract.functions.endBidDate().call())
+    cancellation_end = MayaDT(worklock_agent.contract.functions.endCancellationDate().call())
 
-    # TODO: Include calculated refund and deposit rates
+    bidding_duration = bidding_end - bidding_start
+    cancellation_duration = cancellation_end - bidding_start
+    now = maya.now()
+    bidding_remaining = bidding_end - now if bidding_end > now else timedelta()
+    cancellation_remaining = cancellation_end - now if cancellation_end > now else timedelta()
 
     payload = f"""
 
 Time
 ======================================================
-Start Date ........ {start}
-End Date .......... {end}
-Duration .......... {duration}
-Time Remaining .... {remaining} 
+Bidding Start Date ................... {bidding_start}
+Bidding End Date ..................... {bidding_end}
+Cancellation Window End Date ......... {cancellation_end}
+Bidding Duration ..................... {bidding_duration}
+Cancellation Window Duration ......... {cancellation_duration}
+Bidding Time Remaining ............... {bidding_remaining} 
+Cancellation Window Time Remaining ... {cancellation_remaining} 
+Claiming is available ................ {'Yes' if worklock_agent.is_claiming_available() else 'No'} 
```

Comment:
```suggestion
Claiming available ................ {'Yes' if worklock_agent.is_claiming_available() else 'No'} 
```

Reply:
✔️

Reply:
We should be careful here. Indicating that `Claims available` may make a bidder assume that they have bids that can be claimed. As opposed to `Claiming available` which seems to better apply to the more general window to perform claims.

I think I prefer `Claiming available` or `Claiming phase open`. Thoughts? @KPrasch , @vzotova ?

Reply:
I vote for `Claiming phase open` for the reasons @derekpierre stated. Even better: we consistently use '**Re**claiming/**re**claim' in all worklock files

Reply:
Changed to `Claiming phase open`